ui/src/types/workspace.ts

```diff
@@ -1,11 +1,9 @@
+import type { Workspace as WorkspaceGraphqlType } from "@flow/lib/gql";
+
 import type { Member } from "./member";
 import type { Project } from "./project";
 
-export type Workspace = {
-  id: string;
-  name: string;
-  description?: string;
-  members: Member[] | undefined;
-  projects: Project[] | undefined;
-  personal?: boolean;
+export type Workspace = Pick<WorkspaceGraphqlType, "id" | "name" | "personal"> & {
```

Comment:
Like we were discussing, we want to abstract gql away from code beyond lib/gql. So this should be a lib agnostic type that we move to inside the api code

Reply:
Hmm..yess. As you said above, I'll move the types in the gql folder

Reply:
With the new dir structure, this is fixed

Reply:
Sorry I think you missed a key point. You currently are using a generated gql type as the base for the Workspace type (and you have deleted the Workspace app type inside of src/types/.
This isn't ideal since that means we will be indirectly relying on gql inside the app. 
Instead, we should keep the types/workspace.ts, and inside of the lib/gql api hooks convert the data (which would be of the generated gql Workspace type) to an object that meets the app type Workspace.

Reply:
So you are saying concert from GQL Workspace Type to Workspace type in UI ? 
That makes sense.